Makefile

```diff
@@ -199,18 +199,18 @@ default: build
 
 all: build
 
-# This also rebuilds LinuxMain.swift to include all of the test cases.
-# (The awk script is very fast, so re-running it on every build is reasonable,
-#  but we only update the file when it changes to avoid extra builds.)
-# (Someday, 'swift test' will learn how to auto-discover test cases on Linux,
-# at which time this will no longer be needed.)
-build:
+# This generates a LinuxMain.swift to include all of the test cases.
+# It is needed for all builds before 5.1
```

Comment:
The generate_linux_main values in the workflow don't seem to agree with this comment about 5.1?